bakerydemo/base/fixtures/bakerydemo.json

```diff
@@ -3587,6 +3515,22 @@
       "file_hash": ""
     }
   },
+  {
+    "model": "wagtailusers.userprofile",
```

Comment:
Not sure why this was added, doesn't seem to break anything but can't figure out how I managed to add it in either.

Reply:
It seems that for some reason we didn’t have a Wagtail user profile instance in the fixtures until now. Perhaps this gets created only when accessed the first time – which we now do on every page as part of determining which theme to show the UI in.